packages/common/src/hooks/useDrawer.ts

```diff
@@ -2,15 +2,16 @@ import create from 'zustand';
 import LocalStorage from '../localStorage/LocalStorage';
 
 type DrawerController = {
-  hoverActive: Record<string, boolean>;
+  hoverActive: Record<string, string>;
```

Comment:
now storing the three states of 'hover over', 'hover out' and 🤷 
this allows the sub menu open state to be less directly connected to the hover state